wls-gui-wahllokalsystem/src/components/ergebnisermittlung/TheErgebnisermittlungStimmzettelumschlaegeCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-card-title>Zählen der Stimmzettel</v-card-title>
```

Comment:
`Zählen der Stimmzettel` würde ich als Title für den Menüpunkt in der Navigation sehen, wenn ich es mit dem altsystem vergleiche
```suggestion
      <v-card-title>Wahlurne öffnen und Stimmzettel zählen</v-card-title>
```